src/core/unit.h

```diff
@@ -810,6 +810,7 @@ static inline const UnitVTable* UNIT_VTABLE(const Unit *u) {
 
 Unit* unit_has_dependency(const Unit *u, UnitDependencyAtom atom, Unit *other);
 int unit_get_dependency_array(const Unit *u, UnitDependencyAtom atom, Unit ***ret_array);
+int unit_get_dependency_chain(Unit *u, UnitDependencyAtom atom, Set **ret);
```

Comment:
../src/core/unit.c:6161:5: error: no previous prototype for function 'unit_get_transitive_dependency_set' [-Werror,-Wmissing-prototypes]
int unit_get_transitive_dependency_set(Unit *u, UnitDependencyAtom atom, Set **ret) {

Reply:
Ouch.